ziti/src/main/kotlin/org/openziti/impl/ZitiImpl.kt

```diff
@@ -132,10 +132,11 @@ internal object ZitiImpl : Logged by ZitiLog() {
     }
 
     fun enroll(ks: KeyStore, jwt: ByteArray, name: String): ZitiContext {
-        val enroller = Enroller.fromJWT(String(jwt))
-        val alias = enroller.enroll(null, ks, name)
-
-        return loadContext(ks, alias)
+        TODO("to be removed")
```

Comment:
more context here? i'm left wondering 'why'. if a user calls this they should be told 'why' maybe with a link to an issue etc. also dead code? just remove ?